### PR TITLE
Updated README to be more informative.

### PR DESCRIPTION
**Pull request by dxf**
I found the image using the "Labeled for noncommercial reuse with modification" label on Google Images, so you should be okay using it.

Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+![screenshot](screenshot.png)
 
-This is the game of Connect-4.  The game is played by players taking turns dropping pieces down a column.  The first player to get four consecutive pieces in a row, column, or diagonal wins the game.
+*This is what the game looks like in software.*
 
-Have fun!
+By Leo Chan, (c)1997
 
-Leo 
+*This software is designed to run on BeOS/Haiku operating system. Compatiiblity and stability are not guaranteed with any platform.*
```
Comment (humdingerb) [inline README.md]: I don't think these sentences offer vital information. I suggest removing them and putting "by Leo Chan, ©1997" in italics right under the top heading.

**Review by humdingerb — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dxf) [reply README.md]: i s this ok?

Comment (humdingerb) [reply README.md]: I don't think putting the "by Leo" into a heading is that great. How about simple italics to have it like a by-line to the big heading above it?

Comment (dxf) [reply README.md]: Better? Sorry, my Markdown skills are still in the works ;P

Comment (humdingerb) [reply README.md]: Hmm.. Can't see a difference. it's still a `###` heading...

File: README.md [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
+![game in real life](irl.jpg)
 
-This is the game of Connect-4.  The game is played by players taking turns dropping pieces down a column.  The first player to get four consecutive pieces in a row, column, or diagonal wins the game.
+*This is what Connect 4 looks like in real life. Photo credit: https://www.flickr.com/photos/unloveable/2396019222*
 
-Have fun!
+![screenshot](screenshot.png)
 
-Leo 
+*This is what the game looks like in software.*
```
Comment (scottmc) [inline README.md]: instead of "in software, I'd say "in Haiku"

File: README.md (hunk 1 above)
Comment (dxf) [reply README.md]: https://thanos-taxi.thanos.taxi/i/qpqn.png it did change.

Comment (dxf) [reply README.md]: oh darn i was editing the wrong branch

Comment (dxf) [reply README.md]: solved

File: README.md (hunk 2 above)
Comment (dxf) [reply README.md]: solved

**Review by humdingerb — APPROVED** [review]

**Comment by humdingerb** [comment]
Thanks!

**Pull request merged**